docs/content/getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+1. On the **Basic Information** page, scroll to the "App-Level Tokens" section and click **Generate Token and Scopes**.
+2. Name the token "Development" or something similar and add the `connections:write` scope, then click "Generate".
+3. Save the generated `xapp` token as an environment variable within your project:
+
+```sh
+$ export SLACK_APP_TOKEN=xapp-1-A0123456789-example
+```
+
+The above command works on Linux and macOS but [similar commands are available on Windows](https://superuser.com/questions/212150/how-to-set-env-variable-in-windows-cmd-line/212153#212153).
+
+:::warning[Keep it secret. Keep it safe]
+
+Treat your tokens like a password and [keep it safe](https://docs.slack.dev/authentication/best-practices-for-security). Your app uses these to retrieve and send information to Slack.
+
+:::
+
+A bot token is also needed to interact with the Web API methods as your app's bot user. We can gather this as follows:
+
+1. Navigate to the **OAuth & Permissions** on the left sidebar and install your app to your workspace to generate a token.
+2. After authorizing the installation, you'll return to the **OAuth & Permissions** page and find a **Bot User OAuth Access Token**:
+
+![OAuth Tokens](/img//bot-token.png "Bot OAuth Token")
+
+3. Copy the bot token beginning with `xoxb` from the **OAuth & Permissions page** and then store it in a new environment variable:
+
+```sh
+$ export SLACK_BOT_TOKEN=xoxb-example
+```
+
+After the app has been created, it is time to run it:
+
+```sh
+$ npm run start
+...
+[INFO]  bolt-app ⚡️ Bolt app is running!
+```
+
+With the app running, you can test it out with the following steps:
+
+1. Open a direct message with your app or invite the bot to a public channel.
+2. Send "hello" to the current conversation and wait for a response.
+3. Click the attached button labelled "Click me" to post another reply.
+
+After confirming the app responds, celebrate, then interrupt the process by pressing `CTRL+C` in the terminal to stop your app from running.
+
+</TabItem>
+</Tabs>
+
+## Updating the app
+
+At this point, you're successfully running the getting started Bolt for JavaScript [app](https://github.com/slack-samples/bolt-js-getting-started-app)!
+
+The defaults included leave opportunities abound, so to personalize this app let's now edit the code to respond with a kind farewell.
+
+#### Responding to a farewell
+
+Chat is a common thing apps do and responding to various types of messages can make conversations more interesting.
+
+Using an editor of choice, open the `app.js` file and add the following message listener after the "hello" handler:
+
+```js
+app.message('goodbye', async ({ say }) => {
+  const responses = ['Adios', 'Au revoir', 'Farewell'];
+  const parting = responses[Math.floor(Math.random() * responses.length)];
+  await say(`${parting}!`);
+});
+```
+
+Once the file is saved, restart your app to make sure the latest changes are being used, then go back to channel and say "goodbye".
```

Comment:
suggestion: Since this is a Getting Started guide, I think it's best to hold the developer's hand all the way. I'd suggest that we show the command to run your app rather than having the developer figure out how to "restart your app". This would require a `<TabItem>` for the CLI / Terminal to show the `slack run` and `npm start` commands again and explain to open a DM with their bot and send the "goodbye" text.

Reply:
@mwbrooks Once more a great call that found https://github.com/slackapi/bolt-js/pull/2535/commits/ef6937f48971eb48fbfb29eab0e68539e09ae392! I might be wanting to avoid repetition when writing, but that doesn't translate so well to the guide 😉